system/Validation/Rules.php

```diff
@@ -279,29 +279,50 @@ public function required_with($str = null, ?string $fields = null, array $data =
      *     required_without[id,email]
      *
      * @param string|null $str
+     * @param string|null $otherFields The param fields of required_without[].
+     * @param string|null $field       This rule param fields aren't present, this field is required.
      */
-    public function required_without($str = null, ?string $fields = null, array $data = []): bool
+    public function required_without($str = null, ?string $otherFields = null, array $data = [], ?string $error = null, ?string $field = null): bool
```

Comment:
The method signature change is a breaking change.
We need changelog and ugrading guide.
See
- https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/pull_request.md#changelog
- https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/pull_request.md#upgrading-guide

Reply:
Done.